README.md

```diff
@@ -1,73 +1,47 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [OPTIONAL] Good job 💥. Please also don't forget to add app screenshot to your readme

Reply:
Hi @Rustamxon7, thank you for approving my PR and mentioning this optional request.